projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1388,6 +1388,9 @@ public function get_anchor_podcast() {
 	public function is_difm_lite_in_progress() {
 		if ( function_exists( 'has_blog_sticker' ) ) {
 			return has_blog_sticker( 'difm-lite-in-progress' );
+		} elseif ( function_exists( 'wpcomsh_is_site_sticker_active' ) ) {
+			// For atomic sites
+			return wpcomsh_is_site_sticker_active( 'difm-lite-in-progress' );
```

Comment:
This seems unnecessary for the menu changes, right?

Reply:
Well not technically but the blog sticker which is visible on top of the menu shows the difm lite sticker correctly only when this parameter is returned correctly :).

Reply:
Oh right, I totally missed this. Thanks for clarifying!